core/src/subgraph/registrar.rs

```diff
@@ -606,7 +632,7 @@ fn create_subgraph_version(
         }
         SubgraphVersionSwitchingMode::Synced => {
             // Just set the new version is pending, unless there is no current versions yet
```

Comment:
Looks like I typo'd here, can you `s/is pending/as pending/` while you're in here :joy:

Reply:
Actually, it would help to clarify this comment too with the new behaviour

Reply:
Ok, I rewrote the comment.